pygeoapi/filters.py

```diff
@@ -13,14 +13,17 @@ def combine(sub_filters, combinator="AND"):
         :rtype: filtered dict
     """
     # check for sub_filters being the evaluation result of multiple sub_filters
-    assert isinstance(sub_filters, tuple)
+    if not isinstance(sub_filters, tuple):
+        raise AssertionError("Invalid sub_filters value")
```

Comment:
I would raise a new specific exception for CQL here (`CQLException` or something?), look at some other specific exceptions in the code base

Reply:
Okay I will work on it

Reply:
@FarheenB 
- You have a generic `AssertionError`. The best approach is to have parent class `CQLException` and then chiild clises for the specific error, eg: `CQLExceptionSubFilters` 
@francbartoli : Should we have a module `exceptions.py` and all exception classes would be there described and imported ?

Reply:
@jorgejesus yep that's the idea

Reply:
Done